server/config/email.js

```diff
@@ -1,5 +1,21 @@
 const config = {
-  sendgridApiKey: process.env.SENDGRID_API_KEY
+  sendgridApiKey: process.env.SENDGRID_API_KEY,
+  getSend () {
+    let mail
+    switch (process.env.NODE_ENV) {
+      case 'test':
+      case 'prod':
```

Comment:
my OCD wants you to switch `'test'` and `'prod'` to match the order in other config files :P

Reply:
I checked and the only config with prod and test is the rollbar.js. database.js and this one has the test and prod :) anyway, if that leaves really really happy and with sweet dreams, I can alter both

Reply:
you're right, dang! nevermind haha